### PR TITLE
September Newsletter

### PR DESCRIPTION
**Pull request by LikkanChung**
Please have a read through and merge if you don't find anything wrong. I want to get this out as soon as poss. Thnaks

### CODE REVIEW

File: _posts/2019-09-09-freshers-week-2019.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+Find out more [here](https://localhackday.mlh.io/learn/).
+
+Time and Date: Saturday 12<sup>th</sup> October
+
+Location: TBC, University of Birmingham
+
+### Linux InstallFest (with AFNOM)
+
+We're working with AFNOM to bring you a Linux install workshop. Linux is the preferred operating system of many developers and mnay students aren't quite familair with this yet. This workshop is designed to help you become familiar with Linux and how to start using it.
```
Comment (jedevc) [inline _posts/2019-09-09-freshers-week-2019.md]: > any [many] students arent [super familiar] with it at the beginning of the year/

File: _posts/2019-09-09-freshers-week-2019.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+AFNOM is one of the EPS Societies, and you can find more out about them here: [link](https://afnom.net)
+
+Time and Date: Wednesday 16<sup>th</sup> October
+
+Location: TBC
+
+### Tech Talk
+
+This is our first Tech Talk of the year. Learn something new and we'll bring some free pizza for you. Follow our social channels and keep an eye out for further updates.
```
Comment (jedevc) [inline _posts/2019-09-09-freshers-week-2019.md]: Maybe mention who's doing the talk?

File: _posts/2019-09-09-freshers-week-2019.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+## New Website
+
+Over the summer, Jusin (our vice-president) has been hard at work reworking our website. He's added some great new features such as the [newsletters](https://cssbham.com/newsletter) you're reading now and a [calendar](https://cssbham.com/calendar) where you can find all of our events in one place. 
+
+Have a read of wht Justin has to say about the website in his blog post [here](https://cssbham.com/newsletter/website-update). 
+
+---
+
+Thanks for reading.
```
Comment (jedevc) [inline _posts/2019-09-09-freshers-week-2019.md]: > [Looking forward to seeing you all soon!]

File: _posts/2019-09-09-freshers-week-2019.md (hunk 2 above)
Comment (LikkanChung) [reply _posts/2019-09-09-freshers-week-2019.md]: WIll give more details when Jon confirms

**Pull request merged**